pkg/sql/opt/cat/utils.go

```diff
@@ -122,15 +122,15 @@ func ResolveTableIndex(
 }
 
 // ConvertColumnIDsToOrdinals converts a list of ColumnIDs (such as from a
-// tree.TableRef), to a list of ordinal positions of columns within the given
-// table. See tree.Table for more information on column ordinals.
+// tree.TableRef), to a list of ordinal positions of non-mutation columns within
```

Comment:
It seems like an API bug that this only looks at non-mutation cols. Why not let it handle mutation cols as well, since the API is generic? I don't think any existing caller calls it with mutation cols, so there should be no panics.